lib/vanagon/component/dockerfile.rb

```diff
@@ -66,21 +66,26 @@ def andand_multiline(*commands)
       # @return [String] the contents of the component's Dockerfile.
       def generate
         steps = [
-          unpack_step,
-          patch_step,
-          configure_step,
-          build_step,
-          check_step,
-          install_step,
+          unpack_step(),
+          patch_step(),
+          configure_step(),
+          build_step(),
+          check_step(),
+          install_step(),
         ]
 
         if component.install_only
           steps = [
-            install_step,
+            install_step(),
           ]
         end
 
-        steps.join("\n")
+        copy_sources = [
+          "COPY #{component.name}_sources ${workdir}"
```

Comment:
missing separating commas

Reply:
Oh sorry, this isn't yet ready for review. I just like pushing things up out of habit. I'll comment on here when the new changes are ready for review.